### DISCUSSION TITLE
关于相册和Twikoo评论,有没有可能在下个版本直接集成

### DISCUSSION THREAD

**Discussion by laogou717** [MaLuns/vitepress-theme-async#3] [General]
从 Hexo 转到 VitePress 比我预想的要复杂得多，折腾了半天，壁纸配置就是不显示图片，Twikoo 的评论插槽也没搞定，人已经有点麻了… 

![image](https://github.com/user-attachments/assets/3a2a633a-ae42-498a-8b91-a2702730b3d9)

**Comment by MaLuns** [comment]
是相册页组件没有生效吗? 是不是 globalComponents 没有配置上? 稍后把文档修补下

**Reply by laogou717** [reply]
我配置里是默认把全局变量开了,不过我发现它只在dev模式下无法正常加载,build后显示是正常的

**Reply by MaLuns** [reply]
> 我配置里是默认把全局变量开了,不过我发现它只在dev模式下无法正常加载,build后显示是正常的

有可以还原的小 demo 吗

**Reply by laogou717** [reply]
还没,我就只在本地构建来看看,感觉您这个设计巨好看,特别是墨迹天气的视频头图,简直天才

**Comment by MaLuns** [comment]
twikoo 后面有时间话 集成到主题里吧, 不过现在通过插槽添加, 还是很方便的

**Reply by zhoupeng260641663** [reply]
打扰下，这个插槽怎么使用啊 我想给所有文章添加twikoo

**Reply by MaLuns** [reply]
> 打扰下，这个插槽怎么使用啊 我想给所有文章添加twikoo

可以参考我的，https://github.com/MaLuns/blog/tree/master/.vitepress

**Reply by zhoupeng260641663** [reply]
> > 打扰下，这个插槽怎么使用啊 我想给所有文章添加twikoo
> 
> 可以参考我的，https://github.com/MaLuns/blog/tree/master/.vitepress

好的，感谢大佬！
